.github/workflows/ci.yml

```
@@ -0,0 +1,208 @@
+  # 🚀 JOB 3: DEPLOY - Despliega a EC2 (solo en push, no en PR)
+  deploy:
+    name: 🚀 Deploy to EC2
+    runs-on: ubuntu-latest
+    needs: [test, build]  # ⚠️ DEPENDENCY: Necesita que test y build pasen
+    # 🎯 CONDITIONAL DEPLOYMENT: Solo despliega en push directo (no en PR)
+    if: github.event_name == 'push'
+    
+    steps:
+    # Paso 1: Descargar el paquete creado en el job anterior
+    - name: 📥 Download deployment package
+      uses: actions/download-artifact@v3
```

actionlint (1.7.7)

the runner of "actions/download-artifact@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)